data/virt_autotest/kubevirt_tests_config_parser.pl

```diff
@@ -0,0 +1,21 @@
+use strict;
+use warnings;
+use Config::Tiny;
+
```

Comment:
Please add description for this new file.

Reply:
Added description of this script.